File: src/main/java/com/hungery/hungeryapp/data/Menu.java
package com.hungery.hungeryapp.data;

import javax.persistence.*;

@Entity
@Table(name = "menu")
public class Menu {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long menuid;

    @Column(name = "nimi")
    public String nimi;

    @Column(name = "kuvaus")
    public String kuvaus;

    @Column(name = "hinta")
    public String hinta;

    @Column(name = "kuva")
    public Byte kuva;

    @Column(name = "tuotekategoria")
    public String tuotekategoria;

    @Column(name = "nimiravintola")
    public String nimiravintola;


    public Menu(){}
}

File: frontend/src/components/Menu.js
import React from 'react'
import Tuote from './Tuote'

export default function Menu(props) {

  return (
    <div>
        <div className='menu'>
                { props.menus.map( p => <Tuote nimi={p.nimi} nimiravintola={p.nimiravintola} kuvaus={p.kuvaus} hinta={p.hinta} kuva={p.kuva} qty={p.qty}/>) }    
        </div>
    </div>
  )
}
